docker-entrypoint.sh

```diff
@@ -28,6 +35,7 @@ BASE_URL = '$BASE_URL'
 SECRET_KEY = '$(< /dev/urandom tr -dc A-Za-z0-9_\!\@\#\$\%\^\&\*\(\)-+= | head -c 32)'
 OAUTH2_CLIENT_ID = '$CLIENT_ID'
 OAUTH2_CLIENT_SECRET = '$CLIENT_SECRET'
+PLUGINS = ${PLUGS}
```

Comment:
@frameloss Any chance you can make this change? I'd like to merge this PR. Thanks!